### DISCUSSION TITLE
how to show the users State?

### DISCUSSION THREAD

**Discussion by johndjchan** [yellowtree/geoip-detect#208] [Q&A]
hi there thx for making this plugin, its great

just want to know what is the correct plugin to show the users State in the content
e.g.

This offer is exclusive for [shortcode to show the users state]

many thzx

**Comment by benjaminpick** [comment]
Sorry I missed this question. A "state" is called "subdivision" here.

`[geoip_detect2 property="mostSpecificSubdivision"] `
